infoschema/builder.go

```diff
@@ -91,7 +91,7 @@ func (b *Builder) ApplyDiff(m *meta.Meta, diff *model.SchemaDiff) ([]int64, erro
 		}
 
 		tmpIDs := tblIDs
-		if diff.Type == model.ActionRenameTable && diff.OldSchemaID != diff.SchemaID {
+		if (diff.Type == model.ActionRenameTable || diff.Type == model.ActionRenameTables) && diff.OldSchemaID != diff.SchemaID {
```

Comment:
In rename multiple tables, schema IDs are in AffectedOption, need they be checked?

Reply:
AffectedOption has been checked in the following code. I think maybe we don't need to check it again.